pages/common/git-obliterate.md

```diff
@@ -0,0 +1,13 @@
+# git obliterate
+
+> Remove a file from all history in a Git repository.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-obliterate>.
+
+- Remove all existence of a file including history:
+
+`git obliterate {{file_1 file_2 ...}}`
+
+- Remove existence of a file between 2 commits:
```

Comment:
Perhaps this will read better?
```suggestion
- Remove a file between 2 commits in the current repository's history:
```

Reply:
I think not. It makes it sound a little worse IMO

Reply:
I also prefer the original, however I have one small suggestion:
```suggestion
- Erase the existence of a file between 2 commits:
```